app/controllers/api/v1/merchants_controller.rb

```diff
@@ -17,8 +17,16 @@ def update
   end
 
   def create
-      new_merchant = Merchant.create(merchant_params)
-      render json: MerchantSerializer.new(new_merchant), status: 201
+    begin
+      new_merchant = Merchant.create!(merchant_params)
```

Comment:
I am wondering if we can do the same thing here with the error serializer and not have this in the controller like this? We would need to do some research.